packages/gatsby-theme-docs/src/components/release-notes-subscribe-links.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const ReleaseNotesSubscribeLinks = () => <div>{'TODO subscribe links'}</div>;
```

Comment:
Here we will add the "subscribe" links (rss feed, etc.). To be done in a separate PR.